devel/git-interactive-rebase-tool/Portfile

```diff
@@ -0,0 +1,84 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem       1.0
+PortGroup        github 1.0
+PortGroup        cargo  1.0
+
+github.setup     MitMaro git-interactive-rebase-tool 1.2.1
+
+categories       devel
+platforms        darwin
+supported_archs  noarch
+license          GPL-3
+maintainers      {judaew.com:judaew @judaew} openmaintainer
+
+description      Native cross platform full feature terminal based sequence \
+                 editor for git interactive rebase.
+long_description ${description}
+homepage         https://gitrebasetool.mitmaro.ca
+
+checksums        git-interactive-rebase-tool-${version}.tar.gz \
+                 rmd160 5c9a50a09fe314231973274cdb2dce44b01a5101 \
+                 sha256 b2903c772563843f158f9b009630daa59ce1d463e44ae34ddaa01caf3be66a50 \
+                 size   3626037
+
+depends_lib      port:git port:ncurses
```

Comment:
Use `depends_lib-append` instead to not overwrite portgroup dependencies. Also list each dependency on its own line.